tekton/pipelines/source-container.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: source-container-0-1  # dot is not allowed in the name
+spec:
+  params:
+    - name: OSBS_IMAGE
```

Comment:
I think that we need to decide on standards as early as possible, because things we *don't* decide on will stick just as hard as things we *do* decide on.

Given that; I'd like to see UPPER_UNDER_STYLE reserved for env vars (and for use in e.g. embedded shell scripts), with everything else being lower-kebab-style. @MartinBasti, @rcerven; WDYT?

Reply:
I don't have strong preference, I saw pipelines with both styles for params